components/brave_new_tab_ui/reducers/new_tab_reducer.tsx

```diff
@@ -43,8 +44,13 @@ export const newTabReducer: Reducer<NewTab.State | undefined> = (state: NewTab.S
         ...initialDataPayload.preferences,
         stats: initialDataPayload.stats,
         ...initialDataPayload.privateTabData,
-        topSites: initialDataPayload.topSites
+        topSites: initialDataPayload.topSites,
+        brandedWallpaperData: initialDataPayload.brandedWallpaperData
       }
+      // TODO(petemill): only get backgroundImage if no sponsored background this time.
```

Comment:
(not for this pr): would be cool if we could have these TODO as follow-up issues

Reply:
Often TODO is used not because something needs following up now, but as a note for a developer that they may want to change something if the context changes or the feature expands. In this case, it's pointing out that this is extra work, but it's only a string. Therefore if the background API changes to do more than just simple string selection, then we should change it.

Similarly, the TODO a few lines below about the nested dispatches warns that if bugs are encountered then we should change it.